o1vm/src/pickles/mod.rs

```diff
@@ -16,5 +16,21 @@ pub mod column_env;
 pub mod proof;
 pub mod prover;
 
+/// Maximum degree of the constraints.
+/// It does include the additional degree induced by the multiplication of the
+/// selectors.
+pub const MAXIMUM_DEGREE_CONSTRAINTS: u64 = 6;
+
+/// Degree of the quotient polynomial. We do evaluate all polynomials on d8
+/// (because of the value of [MAXIMUM_DEGREE_CONSTRAINTS]), and therefore, we do
+/// have a degree 7 for the quotient polynomial.
+/// Used to keep track of the number of chunks we do have when we commit to the
+/// quotient polynomial.
+pub const DEGREE_QUOTIENT_POLYNOMIAL: u64 = 7;
```

Comment:
it's not because we evaluate on d8 that the degree is 8
it should be degree of the constraints - 1